wtc.yml

```diff
@@ -0,0 +1,12 @@
+no_trace: false
+trig_async: [run, test]
+rules:
+  - name: changer
+    match: \.go$
+    trig_async:
+      - test
+      - run
+  - name: run
+    command: go run main.go server
+  - name: test
+    command: go test ./...
```

Comment:
You can trigger the tests only for test files changes. Running tests by packages.
```yaml
- name: test
  match: _test\.go$
  command: go test -cover {PKG}
```

Reply:
Yeah looks good I've made that change, I've made it so it runs go test -cover {PKG} if a test file changes otherwise it runs all tests when a normal file changes. It also doesn't restart the server if a test file changes. Let me know if you think thats good